Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -254,6 +254,35 @@ def sudo_purge
     system "/usr/bin/sudo", "--non-interactive", "/usr/sbin/purge"
   end
 
+  def setup_tar_owner_group_args!
+    # Unset the owner/group for reproducible bottles.
+    # Use gnu-tar on Linux
+    return ["--owner", "0", "--group", "0"].freeze if OS.linux?
+
+    bsdtar_args = ["--uid", "0", "--gid", "0"].freeze
+
+    # System bsdtar is new enough on macOS Catalina and above.
+    return bsdtar_args if OS.mac? && MacOS.version >= :catalina
```

Comment:
You can drop the `OS.mac?` check here. `MacOS.version >= :anything` always evaluates to false on Linux. (And `MacOS.version < :anything` is always true.) But keeping it too is fine.

Reply:
@carlocab Nope 😁. On `brew tests --generic` both `OS.mac?` and `OS.linux?` are `false` (and `MacOS` isn't defined). This layer exists to make it easier to potentially port to other OSs in future.

Reply:
Ah right. Generic OS. So often forgotten 😢